content/en/continuous_integration/pipelines/azure.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Set up Tracing on an Azure Pipeline
+kind: documentation
+aliases:
+  - /continuous_integration/setup_pipelines/azure
+further_reading:
+    - link: "/continuous_integration/pipelines"
+      tag: "Documentation"
+      text: "Explore Pipeline Execution Results and Performance"
+    - link: "/continuous_integration/pipelines/custom_commands/"
+      tag: "Documentation"
+      text: "Extend Pipeline Visibility by tracing individual commands"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+    - link: "/continuous_integration/pipelines/custom_tags_and_metrics/"
+      tag: "Documentation"
+      text: "Extend Pipeline Visibility by adding custom tags and metrics"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Compatibility
+
+- **Custom spans**: Configure custom spans
+
+- **Custom pre-defined tags**: Set [custom tags][6] to all generated pipeline and job spans
+
+- **Custom tags and metrics at runtime**: Configure [custom tags][7] and metrics at runtime
```

Comment:
```suggestion
## Compatibility

- **Custom pre-defined tags**: Set [custom tags][6] to all generated pipeline and job spans
```
Only this will be available in the end, through the query param `tags` in the URL